src/apps/LoRaMac/periodic-uplink-lpp/B-L072Z-LRWAN1/main.c

```diff
@@ -172,7 +172,7 @@ void run_main_loop()
 
     while (1)
     {
-        do_n_transmissions(2); // do 2 transmissions on the same network
+        do_n_transmissions(3); // do 2 transmissions on the same network

```

Comment:
The comment should answer for the question "why", the code should answer for the question "what". Here you have in one line to pieces of saying "what" where one denies to the other.